README.md

```diff
@@ -20,50 +20,71 @@ Load in the dependencies and construct some toy data.
 ```julia
 julia> using DataFrames, FeatureTransforms
 
-julia> df = DataFrame(:a=>[1, 2, 3, 4, 5], :b=>[5, 4, 3, 2, 1], :c=>[0, 1, 0, 1, 0])
+julia> df = DataFrame(:a=>[1, 2, 3, 4, 5], :b=>[5, 4, 3, 2, 1], :c=>[2, 1, 3, 1, 3])
 5×3 DataFrame
- Row │ a      b      c
-     │ Int64  Int64  Int64
+ Row │ a      b      c     
+     │ Int64  Int64  Int64 
 ─────┼─────────────────────
-   1 │     1      5      0
+   1 │     1      5      2
    2 │     2      4      1
-   3 │     3      3      0
+   3 │     3      3      3
    4 │     4      2      1
-   5 │     5      1      0
+   5 │     5      1      3
 ```
 
-We construct the transformations that we want to `apply` to the data, which can be non-mutating (`apply`) or mutating (`apply!`) if supported.
-Note that non-mutating transformations do not necessarily return the same type, even when applied to all the elements.
+Next, we construct the `Transform` that we want to `apply` to the data, which can either be non-mutating (`apply`) or mutating (`apply!`).
+All `Transforms` support the non-mutating `apply` method any `Transform` that changes the type or dimension of the input does not support mutation.
+
+In either case, the return will be the same type as the input.
+So if you provide an `Array` you get back an `Array`, and if you provide a `Table` you will get back a `Table`.
```

Comment:
should probably be combined with the previous sentence.

In either case, the return will be the same type as the input, so if you provide ...